checklist.html

```diff
@@ -28,70 +28,35 @@ <h2 id="toc_title" class="toc-title">Categories</h2>
 <section class="resources-section checklist-introduction" id="section-introduction" aria-labelledby="label-introduction">
   <h2 id="label-introduction">Introduction</h2>
   <p>
-    This checklist is build using the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">The Web Content Accessibility Guidelines <abbr>WCAG</abbr></a>, a shared standard for web content accessibility that meets the needs of individuals, organizations, and governments. There are three levels of compliance within WCAG, <strong>A</strong>, <strong>AA</strong>, and <strong>AAA</strong>. These reflect the priority of support:
+    This checklist uses the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">The Web Content Accessibility Guidelines (<abbr>WCAG</abbr>)</a>, , a shared standard for web content accessibility for individuals, organizations, and governments. There are three levels of compliance which reflect the priority of accessibility support:
   </p>
   <ol>
-    <li><strong>A</strong>: Must support.</li>
-    <li><strong>AA</strong>: Should support.</li>
-    <li><strong>AAA</strong>: May support.</li>
+    <li><strong>A</strong>: Essential. If this isn't met, assistive technology cannot read, understand, or operate the technology.</li>
+    <li><strong>AA</strong>: Good support. Now required for EU government and public body websites.</li>
+    <li><strong>AAA</strong>: High level support. This is typically reserved for sites that serve a specialized audience.</li>
   </ol>
   <p>
-    While <strong>A</strong> support is mandatory, most websites should have a level of <strong>AA</strong> support to best guarantee support. <strong>AAA</strong> level compliance is typically reserved for sites that serve a specialized audience. It should also be noted that the different levels of WCAG support do not necessarily indicate an increased level of difficulty to implement.
-  </p>
-  <p>
-    This checklist is built using WCAG as a reference point. It targets many, but not all level <strong>AA</strong> issues.
+    Note that the different levels of WCAG support do not necessarily indicate an increased level of difficulty to implement. This checklist is built using WCAG as a reference point. It targets many, but not all level AA issues.
```

Comment:
So, "success criterion" is what the WCAG uses, but it's a jargon term that's kind of difficult to grok at first pass. Should we be consistent with honoring the WCAG terminology, or explain what it means and use our own terminology?

Reply:
Understood, but my comment was about the fact that right now "success criterion" is used in the prose as well as "issues" so we should be consistent.

Personally though, per your response, I'm more in favor of educating people to what the term means and then using it, rather than flat out shielding people from having to learn something.

Reply:
Pushed up some copy tweaks for the intro section. How does it read to you now?